libcontainer/cgroups/fs/cpuset_test.go

```diff
@@ -3,11 +3,148 @@
 package fs
 
 import (
+	"fmt"
+	"reflect"
 	"testing"
 
+	"github.com/opencontainers/runc/libcontainer/cgroups"
 	"github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
 )
 
+const (
+	cpus                  = "0-2,7,12-14\n"
+	cpuExclusive          = "1\n"
+	mems                  = "1-4,6,9\n"
+	memHardwall           = "0\n"
+	memExclusive          = "0\n"
+	memoryMigrate         = "1\n"
+	memorySpreadPage      = "0\n"
+	memorySpeadSlab       = "1\n"
+	memoryPressure        = "34377\n"
+	schedLoadBalance      = "1\n"
+	schedRelaxDomainLevel = "-1\n"
+)
+
+type testCase struct {
+	desc               string
+	filename, contents string
+	removeFile         bool
+}
+
+var (
+	cpusetTestFiles = map[string]string{
+		"cpuset.cpus":                     cpus,
+		"cpuset.cpu_exclusive":            cpuExclusive,
+		"cpuset.mems":                     mems,
+		"cpuset.mem_hardwall":             memHardwall,
+		"cpuset.mem_exclusive":            memExclusive,
+		"cpuset.memory_migrate":           memoryMigrate,
+		"cpuset.memory_spread_page":       memorySpreadPage,
+		"cpuset.memory_spread_slab":       memorySpeadSlab,
+		"cpuset.memory_pressure":          memoryPressure,
+		"cpuset.sched_load_balance":       schedLoadBalance,
+		"cpuset.sched_relax_domain_level": schedRelaxDomainLevel,
+	}
+	testCases = []testCase{
```

Comment:
You are introducing a global variable named `testCases` here and you only use it in a single function. I think this should either be renamed or moved to that function.

Reply:
renamed